### DISCUSSION TITLE
error on installation

### DISCUSSION THREAD

**Discussion by pieterlu** [imajaydwivedi/SQLDBATools#50] [Q&A]
I tried installing on one of our servers using the command "Install-Module -Name SQLDBATools", but received the following warning messages:

WARNING: The externally managed, dependent module 'dbatools' is not installed on this computer. To use the current module 'SQLDBATools', ens
ure that its dependent module 'dbatools' is installed.
WARNING: The externally managed, dependent module 'ImportExcel' is not installed on this computer. To use the current module 'SQLDBATools', 
ensure that its dependent module 'ImportExcel' is installed.
WARNING: The externally managed, dependent module 'PoshRSJob' is not installed on this computer. To use the current module 'SQLDBATools', en
sure that its dependent module 'PoshRSJob' is installed.
WARNING: The externally managed, dependent module 'EnhancedHTML2' is not installed on this computer. To use the current module 'SQLDBATools'
, ensure that its dependent module 'EnhancedHTML2' is installed.

**Comment by imajaydwivedi** [comment]
Hi @pieterlu , In this module, I am using dbatools, ImportExcel, PoshRSJob modules also. 

So, kindly execute below command to install above modules first.

```
Install-Module dbatools, ImportExcel, PoshRSJob, EnhancedHTML2
Install-Module SQLDBATools
```
